cookbook/integration_litellm.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
<p>can you add links to the 2 traces in langfuse, I can screenshot them and add to this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>